ArduPlane/quadplane.cpp

```diff
@@ -762,10 +762,6 @@ bool QuadPlane::setup(void)
 
     motors->init(frame_class, frame_type);
```

Comment:
Could we add a comment here noting that `init()` can actually fail and leave the motors initialised, please?

Reply:
That is also true if you have selected a frame type that the particular class does not have, not a issue that was added with the scripting stuff.